montrek-software/montrek
# Debug history

# Solution

- The history mechanisms works by collecting all change dates of a set of satellites connected to a hub, loops over those, sets the reference_date and call `std_queryset`
- In company referecne_date was set in `std_queryset` as being now, hence no changes were seen 

# How to test

- Change a company's attribute and observe the history

![image](https://github.com/chrishombach/montrek/assets/29869670/03890ce2-af6d-4d8c-a1ac-a93bce120406)
